docs/topics/gradle/gradle-compiler-options.md

```diff
@@ -167,26 +167,41 @@ compileKotlin {
 
 | Name | Description | Possible values |Default value |
 |------|-------------|-----------------|--------------|
-| `apiVersion` | Restrict the use of declarations to those from the specified version of bundled libraries | "1.3" (DEPRECATED), "1.4" (DEPRECATED), "1.5", "1.6", "1.7" |  |
-| `languageVersion` | Provide source compatibility with the specified version of Kotlin | "1.4" (DEPRECATED), "1.5", "1.6", "1.7" |  |
+| `apiVersion` | Restrict the use of declarations to those from the specified version of bundled libraries | "1.3" (DEPRECATED), "1.4" (DEPRECATED), "1.5", "1.6", "1.7", "1.8", "1.9" |  |
+| `languageVersion` | Provide source compatibility with the specified version of Kotlin | "1.4" (DEPRECATED), "1.5", "1.6", "1.7", "1.8", "1.9" |  |
+
+Also, see [Types for compiler options](#types-for-compiler-options).
```

Comment:
I think once we will publish KGP api - we could drop this page or, at least, this section with compiler arguments values

Reply:
Ok, I'll write it down.

Reply:
https://youtrack.jetbrains.com/issue/KT-55608/Docs-Update-docs-after-KGP-API-documentation